tutorials/run-koajs-on-google-app-engine.md

```diff
@@ -35,11 +35,11 @@ Platform.
 
 Create a `server.js` file with the following contents:
 
-    const koa = require('koa');
-    const app = koa();
+    const Koa = require('koa');
+    const app = new Koa();
 
-    app.use(function *() {
-      this.body = 'Hello World!';
+    app.use(ctx => {
+      ctx.body = 'Hello World';
```

Comment:
Please restore the exclamation point so that the text that follows the code sample is accurate.